### ISSUE TITLE
Music Album Duration

### ISSUE THREAD

**Issue by lizardkg** [schemaorg/schemaorg#3443]
Hello, what's the rationale for the lack of duration in MusicAlbums? If something defines an album is its length, but no duration is available unless tagged as MusicRelease. There is timeRequired, but it is also described as "approximate or typical time it usually takes to work with or through the content", but music album lengths are anything but approximates. How do you tag duration in music album without having to markup a Music Release?

**Comment by VladimirAlexiev** [comment]
Hi @lizardkg !
- It's the Work-Expression-Manifestation-Item (WEMI) distinction of FRBR
MusicAlbum corresponds to Expression, while MusicRelease corresponds to Manifestation.
I can imagine that the same album can be released several times (eg vynil vs CD vs SoundCloud; or original vs Anniversary vs Gold/Collectors edition) and the timing between them may vary a bit
- But if you really want to use "duration" on MusicAlbum, nobody can stop you. 
The list of props on that page is made of `schema:domainIncludes`, which is descriptive but not prescriptive.
In other words, it suggests on which classes to use the property, but doesn't preclude its use on other classes.

HTH!
